### DISCUSSION TITLE
EHFTV Downloads Not Working

### DISCUSSION THREAD

**Discussion by dgmulligan** [aclap-dev/video-downloadhelper#200] [Help and Q&A]
### Your question

Running the latest version along with the companion.

I've tried downloading from the following address:
https://ehftv.com/player/vod/france-vs-sweden?entryId=0_zh6w3a5i&competitions=Men%27s+EHF+EURO
https://ehftv.com/player/vod/germany-vs-denmark?entryId=0_r7n8gpy0&competitions=Men%27s%EHF%EURO

VDH recognizes the ad that comes before the match, and will download that, but nothing after the ad. I've tried reloading the app after the add when the match is playing but it doesn't recognize it.

I'm running Chrome Version 121.0.6167.185 (Official Build) (64-bit).

### Addon & CoApp version and details

Video DownloadHelper 8.2.1.1 (stable)
Video DownloadHelper
Version 8.2.1.1 (stable)
Browser locale: en-US
Production build
Built on 2024-02-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google
⊖Platform
Platform Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
⊖Companion App installed
Found companion app: VdhCoApp 2.0.19

**Comment by paulrouget** [comment]
Can you try this:

- Settings
- More
- Video Qualities
- uncheck "ignore low quality videos"

Tell me if you see more download options.

**Reply by dgmulligan** [reply]
Tried it.

It's still popping up with just the option to download the advertisement.

I let it run for a minute and then the advertisement download option disappears and I get "No media to process in the current tab" even though the video is running.

**Comment by paulrouget** [comment]
I created an account. It seems to work well for me. But maybe because I use an adblocker.

Do you see this?

<img width="500" alt="Screenshot 2024-02-22 at 14 57 24" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/e7c7b1ba-aec5-4c8c-a884-c16445010dc9">

**Reply by paulrouget** [reply]
I'm using Version 8.2.1.1, on Chrome, without an adblocker, and I can download without any issue.

**Reply by dgmulligan** [reply]
No, I never received the HLS streaming option for any video resolutions. All I received was the option to download the ad. 

I'm not using an adblocker so it definitely seems like the ad is what's tripping up VDH.

It's strange because I'm almost positive I had no problem from this site 2 years ago.

**Reply by paulrouget** [reply]
Even if I do not use an adblocker, I still see the HLS option. So I'm not sure how this would not work for you.

Hmm… can you make sure to play at least ~10 seconds of the match before checking if the video is listed?

**Reply by Mattmjs** [reply]
@dgmulligan , maybe you should try with an adblocker or adguard dns.

**Reply by dgmulligan** [reply]
@paulrouget 

1. Click on link to watch video
2. Ad plays
3. Match begins playing, wait 90 seconds
4. Check VDH
5. Only option is to download ad
6. Run analyze page
7. Gallery option shows up

But still none of the actual match streams.
![90 seconds playing](https://github.com/aclap-dev/video-downloadhelper/assets/160822815/d38992fa-7dd8-444d-92ab-a6421a5bf488)
![After analyze](https://github.com/aclap-dev/video-downloadhelper/assets/160822815/c8f36144-85c0-4406-bc98-3e121544f13a)

**Reply by dgmulligan** [reply]
@Mattmjs 
Yeah, seems like that might be the only answer.

I don't like to throw on adblockers because I have friends who work for online platforms and rely on ad revenue.

Aw well.
